extensions/oidc/runtime/src/main/java/io/quarkus/oidc/runtime/OidcUtils.java

```diff
@@ -423,4 +425,27 @@ public static String decryptString(String jweString, SecretKey key) throws Excep
         jwe.setCompactSerialization(jweString);
         return jwe.getPlaintextString();
     }
+
+    public static String[] parseCookieValue(String value) {
+        if (value.contains(CRLF)) {
```

Comment:
We should not be guarding against this here. This should just be prohibited at the Netty encoder level, and AFAIK it already is.

The place to block these sort of issues is at the HTTP level, not piecemeal all through the code.

Reply:
@stuartwdouglas Good point, I was not aware of it. I've confirmed the CRLF sequence is replaced by `??` by Netty. Ideally the error would be caused - it just masks it but with `??` there is nothing that we can do.

Reply:
Nope, it was a bad test

Reply:
After spending more time with wire tracing I think I can confirm that the cookies with CRLF sequences are not even available in the routing context cookies map, which is good.